oauthproxy.go

```diff
@@ -112,7 +114,7 @@ type OAuthProxy struct {
 	jwtBearerVerifiers   []*oidc.IDTokenVerifier
 	compiledRegex        []*regexp.Regexp
 	templates            *template.Template
-	realClientIPParser   realClientIPParser
+	realClientIPParser   logging.RealClientIPParser
```

Comment:
Use of "real" IP is not just a logging concern.

Reply:
As far as I'm aware, we currently only use it within logging. Though I appreciate the sentiment. I would suggest we leave it as is now and then move it if we ever find another use case for it.